packages/lesswrong/components/common/Header.tsx

```diff
@@ -289,6 +504,134 @@ const Header = ({
       <UsersMenu />
     </AnalyticsContext>
   </div>
+  
+  // the items on the right-hand side (search, notifications, user menu, login/sign up buttons)
+  const rightHeaderItemsNode = <div className={classes.rightHeaderItems}>
+    <NoSSR onSSR={<div className={classes.searchSSRStandin} />} >
+      <SearchBar onSetIsActive={setSearchOpen} searchResultsArea={searchResultsArea} />
+    </NoSSR>
+    {!isEAForum && usersMenuNode}
+    {!currentUser && <UsersAccountMenu />}
+    {currentUser && !currentUser.usernameUnset && <KarmaChangeNotifier
+      currentUser={currentUser}
+      className={(isEAForum && searchOpen) ? classes.hideXsDown : undefined}
+    />}
+    {currentUser && !currentUser.usernameUnset && <NotificationsMenuButton
+      unreadNotifications={unreadNotifications}
+      toggle={handleNotificationToggle}
+      open={notificationOpen}
+      className={(isEAForum && searchOpen) ? classes.hideXsDown : undefined}
+    />}
+    {isEAForum && usersMenuNode}
+  </div>
+  
+  // the left side nav menu
+  const navigationDrawer = <NavigationDrawer
+    open={navigationOpen}
+    handleOpen={() => setNavigationOpen(true)}
+    handleClose={() => setNavigationOpen(false)}
+    toc={toc?.sectionData ?? null}
+  />
+  
+  // the right side notifications menu
+  const notificationsMenu = currentUser && <NotificationsMenu
+    unreadPrivateMessages={unreadPrivateMessages}
+    open={notificationOpen}
+    hasOpened={notificationHasOpened}
+    setIsOpen={handleSetNotificationDrawerOpen}
+  />
+  
+  // special case for the homepage header of EA Forum Giving Season 2023
+  // TODO: delete after 2023
+  const now = moment()
+  const isGivingSeason = isEAForum && moment(timelineSpec.start).isBefore(now) && moment(timelineSpec.end).isAfter(now)
+  if (isGivingSeason && pathname === '/') {
+    return (
+      <AnalyticsContext pageSectionContext="header">
```

Comment:
Is there another analytics context prop that it makes sense to pass?

Reply:
I'm not sure? There's a few links in this header and we can track them as normal link click events. @agnesstenlund if there's anything else you want us to track let me know

Reply:
I mean like, maybe a context for "this is the giving season header". If your reply is "I mean, you could just use the date and path to figure that out", then I agree, but I think it might be easier to do an analytics query if there was one property you could filter on.

Reply:
There's no obvious candidate in `packages/lesswrong/lib/analyticsEvents.tsx:109`, but it's trivial to add one.

Reply:
Oh I mostly meant that we'll know what links we want to track so we could just filter on `to`. But seems reasonable to add another prop so I'll do that.